src/email-sent.js

```diff
@@ -32,20 +32,28 @@ module.exports = function (request) {
     return `La url ${url} non e' valida`;
   }
 
+
+  const mailServerConfig = JSON.parse((process.argv.includes('prod')) ? 
+    fs.readFileSync('config-prod.json').toString('utf8') : 
+    fs.readFileSync('config-dev.json').toString('utf8'));
+
+  const configAccountString = fs.readFileSync('account-config.json').toString('utf8');
+  const accountConfig = JSON.parse(configAccountString);
+
   // Generate test SMTP service account from ethereal.email
   // Only needed if you don't have a real mail account for testing
   nodemailer.createTestAccount((err, account) => {
```

Comment:
Se abbiamo avviato l'applicazione in produzione, non deve essere creato un account di test con `nodemailer.createTestAccount`.